src/age-keygen/devcontainer-feature.json

```
@@ -1,6 +1,6 @@
 {
     "id": "age-keygen",
-    "version": "1.0.15",
+    "version": "1.2.1",
```

```suggestion
    "version": "1.0.16",
```

This is version of the feature which is independent from version of the tool, so let's just bump the patch version